### PR TITLE
Vulpkanin Tail Wagging

### PR DESCRIPTION
**Pull request by FoxxoTrystan**
# Description

Add the ability for Vulpkanins to wag their tails like Lizards and add a far better icon for tail wagging making it not look like only action.

---

<details><summary><h1>Media</h1></summary>
<p>

image
image

</p>
</details>

---

# Changelog

:cl:
- add: Vulpkanins can wag their tails now

### CODE REVIEW

File: Resources/Locale/en-US/deltav/markings/vulpkanin.ftl [hunk 1]
```diff
@@ -251,4 +229,4 @@ marking-VulpHairSpike = Spike
 marking-VulpFacialHairRuff = Ruff
 marking-VulpFacialHairElder = Elder
 marking-VulpFacialHairElderChin = Elder Chin
-marking-VulpFacialHairKita = Kita
+marking-VulpFacialHairKita = Kita
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/deltav/markings/vulpkanin.ftl:232]:
```suggestion
marking-VulpFacialHairKita = Kita

```

File: Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml:1]: Why'd you remove the Vulpkanin species restrictions on these?

File: Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml [hunk 2]
```diff
@@ -866,4 +884,4 @@
   speciesRestriction: [Vulpkanin]
   sprites:
     - sprite: DeltaV/Mobs/Customization/Vulpkanin/facial_hair.rsi
-      state: kita
+      state: kita
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml:887]:
```suggestion
      state: kita

```

File: Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml [hunk 3]
```diff
@@ -97,6 +97,7 @@
       Female: FemaleVulpkanin
       Unsexed: MaleVulpkanin
   - type: DogVision
+  - type: Wagging
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:100]: Is that really the component name? Stupid.

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Customization/Markings/vulpkanin.yml:1]: Having them removed mean they cannot be selected in the creation menu and they should not be selected by default as "Animated".

File: Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml (hunk 3 above)
Comment (DangerRevolution) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:100]: Real.

**Review by DangerRevolution — APPROVED** [review]

**Review by VMSolidus — APPROVED** [review]

**Comment by DangerRevolution** [comment]
Too scared to merge after dismissing death's review, but PR looks good to merge

**Pull request merged**